internal/services/desktopvirtualization/virtual_desktop_host_pool_resource.go

```diff
@@ -139,6 +139,53 @@ func resourceVirtualDesktopHostPool() *pluginsdk.Resource {
 				Default: string(hostpool.PreferredAppGroupTypeDesktop),
 			},
 
+			"scheduled_agent_updates_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"scheduled_agent_updates_timezone": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  "UTC",
+			},
+
+			"scheduled_agent_updates_use_session_host_timezone": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"agent_updates_schedule": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 2,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"day_of_week": {
+							Type:     pluginsdk.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								string(hostpool.DayOfWeekMonday),
+								string(hostpool.DayOfWeekTuesday),
+								string(hostpool.DayOfWeekWednesday),
+								string(hostpool.DayOfWeekThursday),
+								string(hostpool.DayOfWeekFriday),
+								string(hostpool.DayOfWeekSaturday),
+								string(hostpool.DayOfWeekSunday),
+							}, false),
+						},
+
+						"hour_of_day": {
+							Type:         pluginsdk.TypeInt,
+							Required:     true,
+							ValidateFunc: validation.IntBetween(0, 23),
+						},
+					},
+				},
+			},
```

Comment:
could we put this all in a block to clean up the schema? it also will match the API better